theory.md

```diff
@@ -61,3 +61,5 @@ git clone <repository url> [directory]
 3.	How do you check the status of your Git repository?  
 git status 

```

Comment:
Remind me to talk about Markdown syntax for code blocks. This way, your presentation will be a little better when writing commands